src/main/scala/ir/package.scala

```diff
@@ -0,0 +1,3 @@
+package object ir {
+
+}
```

Comment:
What's the point of this empty package object?

Reply:
Don't know it must be a mistake or generated by intellij